boxsdk/object/item.py

```diff
@@ -725,3 +744,20 @@ def remove_classification(self):
             `bool`
         """
         return self.metadata('enterprise', self._classification_template_key).delete()
+
+    @staticmethod
+    def validate_item_id(item_id):
+        """
+        Validates an item ID is a digit
```

Comment:
shouldn't this be numeric, not a single digit? Otherwise we are only able to have 10 item_ids.

Reply:
```suggestion
        Validates an item ID is numeric
```

Reply:
@kingsawyer Great comment, thanks!  FYI for comments like this, a "GitHub Suggestion Comment" is very helpful because it is so easy to commit!